composer/core/state.py

```diff
@@ -764,6 +764,8 @@ def fsdp_device_mesh(self):
         if self.fsdp_enabled:
             if not hasattr(self.model, 'model'):
                 return None
+            if not hasattr(self.model.model, '_device_mesh'):
+                return None
```

Comment:
```suggestion
            if not hasattr(self.model, 'model') or not hasattr(self.model.model, '_device_mesh'):
                return None
```
Minor nit (I understand I gave the suggestion :P)

Reply:
lol

Reply:
but then CI has to re-run